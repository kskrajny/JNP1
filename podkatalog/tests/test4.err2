Error in line 13: ? a_^ 1 b 2 c
Error in line 16: ? b 01 b
