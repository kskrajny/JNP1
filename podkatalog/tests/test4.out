! bilet    SPACJEEEEE     ; bilet    SPACJEEEEE     
! bilet    SPACJEEEEE     ; biletA
! biletD
! biletD
! bilet    SPACJEEEEE     ; biletD
8
